apps/chatbot/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM node:18-alpine
```

```suggestion
FROM node:18.15.0-alpine
```
Just to match what's defined in our `.nvmrc`